ディスク作成時にIsSSHKeysEphemeralを指定するとエラー

SSHKeysを指定、かつIsSSHKeysEphemeralをtrueにすると以下のエラーとなる。

```console
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x16ab1a0]

goroutine 9 [running]:
github.com/sacloud/iaas-service-go/disk/builder.(*FromDiskOrArchiveBuilder).Build(0xc000a0cb60, {0x1ca50d8, 0xc0003c5680}, {0xc000382590, 0x4}, 0x17a1216)
	/Users/kaz/go/pkg/mod/github.com/sacloud/iaas-service-go@v1.1.3/disk/builder/builder.go:343 +0xe0
github.com/sacloud/iaas-service-go/server/builder.(*Builder).Build(0xc0001490e0, {0x1ca50d8, 0xc0003c5680}, {0xc000382590, 0xe8})
	/Users/kaz/go/pkg/mod/github.com/sacloud/iaas-service-go@v1.1.3/server/builder/builder.go:244 +0x447
github.com/sacloud/iaas-service-go/server.(*Service).ApplyWithContext(0xc00052fd88, {0x1ca50d8, 0xc0003c5680}, 0xc00039a0f0)
	/Users/kaz/go/pkg/mod/github.com/sacloud/iaas-service-go@v1.1.3/server/apply_service.go:41 +0x7e
github.com/sacloud/iaas-service-go/server.(*Service).CreateWithContext(0x2385840, {0x1ca50d8, 0xc0003c5680}, 0xc000a08000)
	/Users/kaz/go/pkg/mod/github.com/sacloud/iaas-service-go@v1.1.3/server/create_service.go:32 +0x238
github.com/sacloud/usacloud/pkg/services/iaas.init.37.func3({0x1cbaf58, 0xc0003c5680}, {0x1b01b00, 0xc0009e0380})
	/Users/kaz/go/src/github.com/sacloud/usacloud/pkg/services/iaas/server_services_gen.go:93 +0xdc
github.com/sacloud/usacloud/pkg/core.(*Command).exec.func1.1()
	/Users/kaz/go/src/github.com/sacloud/usacloud/pkg/core/command.go:498 +0x37
github.com/sacloud/usacloud/pkg/core.(*Progress).Exec(0xc0009eb680, 0xc0009e9e90)
	/Users/kaz/go/src/github.com/sacloud/usacloud/pkg/core/progress.go:59 +0x6f
github.com/sacloud/usacloud/pkg/core.(*Command).exec.func1({0x1cbaf58, 0xc0003c5680}, {0x1b01b00, 0xc0009e0380})
	/Users/kaz/go/src/github.com/sacloud/usacloud/pkg/core/command.go:497 +0x14c
github.com/sacloud/usacloud/pkg/core.(*Command).execParallel.func2({0x1cbaf58, 0xc0003c5680})
	/Users/kaz/go/src/github.com/sacloud/usacloud/pkg/core/command.go:549 +0x93
created by github.com/sacloud/usacloud/pkg/core.(*Command).execParallel
	/Users/kaz/go/src/github.com/sacloud/usacloud/pkg/core/command.go:543 +0x3e9

```

--- a/disk/builder/builder.go
+++ b/disk/builder/builder.go
@@ -101,7 +101,7 @@ func (d *FromUnixBuilder) Build(ctx context.Context, zone string, serverID types
 	}
 
 	if d.EditParameter != nil {
-		if d.EditParameter.IsSSHKeysEphemeral {
+		if d.EditParameter.IsSSHKeysEphemeral && d.generatedSSHKey != nil {
 			if err := d.Client.SSHKey.Delete(ctx, d.generatedSSHKey.ID); err != nil {
 				return nil, err
 			}
@@ -339,7 +339,7 @@ func (d *FromDiskOrArchiveBuilder) Build(ctx context.Context, zone string, serve
 	}
 
 	if d.EditParameter != nil {
-		if d.EditParameter.IsSSHKeysEphemeral {
+		if d.EditParameter.IsSSHKeysEphemeral && d.generatedSSHKey != nil {
 			if err := d.Client.SSHKey.Delete(ctx, d.generatedSSHKey.ID); err != nil {
 				return nil, err
 			}

